yarn.lock

```diff
@@ -2447,6 +2447,11 @@
     minimatch "^3.1.2"
```

Comment:
If you run `yarn` from the top-level repo, it should clear up the `moment` and `@tryghost/logging` changes - there's some weird yarn thing where it ignores resolutions if you run it from a workspace package